go.mod

```diff
@@ -1,57 +1,54 @@
 module github.com/CircleCI-Public/circle-policy-agent
 
-go 1.20
+go 1.23.1
 
 require (
-	github.com/open-policy-agent/opa v0.54.0
-	github.com/pmezard/go-difflib v1.0.0
-	github.com/stretchr/testify v1.8.3
-	golang.org/x/exp v0.0.0-20230713183714-613f0c0eb8a1
+	github.com/open-policy-agent/opa v0.68.0
+	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
+	github.com/stretchr/testify v1.9.0
+	golang.org/x/exp v0.0.0-20240909161429-701f63a606c0
```

Comment:
wrt @moses-001 comment - I think if we replace the other usages we could avoid this import completely